### DISCUSSION TITLE
Show automatic discount in webshop

### DISCUSSION THREAD

**Discussion by DHD-Harper** [pretix/pretix#4575] [Ideas]
It proved to be very practical that automatic discounts can be set up on a time-controlled basis. Unfortunately, the customer only sees this when they add the respective product to the shopping cart.

It would be nice if the current discount can be displayed on the webshop landing page as well.

**Comment by raphaelm** [comment]
Yeah, for this reason we currently only recommend the discount feature for quantity-/value-dependent discounts, which they have been originally designed for.

Still, would be nice to support this, it might be complex to implement without performance overhead, though.
